be/src/service/service_be/starrocks_be.cpp

```diff
@@ -212,7 +212,7 @@ void start_be(const std::vector<StorePath>& paths, bool as_cn) {
         options.num_threads = config::brpc_num_threads;
     }
     const auto lake_service_max_concurrency = config::lake_service_max_concurrency;
-    const auto service_name = "starrocks.lake.LakeService";
+    const auto service_name = "starrocks.LakeService";
```

Comment:
will this rename causes transmit failures during the cluster upgrade? e.g. fe still stays at the old protobuf, but BE uses the new protobuf

Reply:
No, I've tested this scenario.